ui/app/AppLayouts/Wallet/WalletLayout.qml

```diff
@@ -213,12 +213,6 @@ Item {
         }
     }
 
-    SeedPhraseBackupWarning {
```

Comment:
A second seedphrase backup warning banner? Removed too although I have a feeling this has never worked properly since I've never seen it before here :)